src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/api/utils/Inputvalidering.kt

```diff
@@ -15,8 +15,8 @@ fun validerBakgrunnsinformasjon(tekst: String?) = tekst?.let {
 }
 
 fun validerBegrunnelse(tekst: String?) = tekst?.let {
```

Comment:
Brukes denne fortsatt?

Reply:
Hvis den brukes for å validere "annet" så kan vi kanskje endre litt på navnet? :)

Reply:
Nei, men den den kan brukes til å validere innholdet ANNET, som egentlig var tanken bak denne men jeg var nok litt forvirret da jeg laget den. 🤷  Kan slette den og...

Reply:
Jeg trodde jeg endret navnet... Men jeg ser det har blitt endret tilbake 😅  skal endre.